### DISCUSSION TITLE
Trouble with installation--no kubeconfig.yaml generated

### DISCUSSION THREAD

**Discussion by ElanVital70** [bionic-gpt/bionic-gpt#471] [General]
I followed the site's instructions for installation but got this error in the final step, to install the application into k3s:

 bionic install
Error: Failed to infer configuration: failed to infer config: in-cluster: (failed to read an incluster environment variable: environment variable not found), kubeconfig: (failed to read kubeconfig from '"/home/curt/.kube/config"': No such file or directory (os error 2))

Caused by:
    0: failed to infer config: in-cluster: (failed to read an incluster environment variable: environment variable not found), kubeconfig: (failed to read kubeconfig from '"/home/curt/.kube/config"': No such file or directory (os error 2))
    1: failed to read kubeconfig from '"/home/curt/.kube/config"': No such file or directory (os error 2)
    2: No such file or directory (os error 2)

I went to k3s documentation on installation and it said that running their script would generate a kubeconfig.yaml. I did so and the file was generated, but the bionic install command looks for it in another location. I was ready to copy it there, but the AI I consulted for tech support said I shouldn't, for this reason:

No, you shouldn't directly copy the k3s.yaml file to your home directory (/home/curt/.kube/config). While the file contains your kubeconfig information, it's not in the standard format expected by the Bionic GPT CLI and other Kubernetes tools.

How can I continue from here?

**Comment by ianpurton** [comment]
Hi you could try our new easier install https://bionic-gpt.com/docs/community-edition/docker-compose/
